src/GMO/Payment/CreditCardPayment.php

```diff
@@ -362,4 +362,26 @@ private function convertRecurringResultFileToArray(string $csvData, string $meth
 
         return $entries;
     }
+
+    public function tds2Auth(string $accessID, string $accessPass, string $tds2Param)
+    {
+        $api = $this->createApiObject();
+
+        $api->setParam('AccessID', $accessID);
+        $api->setParam('AccessPass', $accessPass);
+        $api->setParam('Tds2Param', $tds2Param);
```

Comment:
`AccessID`, `AccessPass`, and `Tds2Param` must begin with a lowercase character.